tests/scaletest/lib/ingress.go

```diff
@@ -333,15 +349,49 @@ func DeleteIngress(namespace string, listOfIngressToDelete []string) ([]string,
 	return listOfDeletedIngresses, nil
 }
 
-func ListIngress(t *testing.T, namespace string) error {
-	t.Logf("Listing ingress in namespace %q:\n", namespace)
+func UpdateIngress(namespace string, listOfIngressToUpdate []string) ([]string, error) {
+	for i := 0; i < len(listOfIngressToUpdate); i++ {
```

Comment:
It will do a length calculation everytime, can we do it before hand and store it in a variable and use that variable here instead?

Reply:
Done!